### PR TITLE
feat(integrationCenter): add integration center module

### PR DESCRIPTION
**Pull request by huxiangtao**
**What this PR does / why we need it**:

add web ui for integration center

image
image
image

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @qiuxiaolian @caosm @artecher 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: web/src/components/integration/component/List.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      {
+        title: intl.get('integration.integrationTime'),
+        dataIndex: 'time',
+        key: 'time',
+      },
+    ];
+    return (
+      <div>
+        <div className="head-bar">
+          <h2>数据源</h2>
```
Comment (caosm) [inline web/src/components/integration/component/List.js]: i18n

File: web/src/components/integration/component/List.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+          <Button type="primary" onClick={this.addDataSource}>
+            {intl.get('operation.add')}
+          </Button>
+        </div>
+        <Table columns={columns} dataSource={integration.integrationList} />
+        <Modal
+          ref="modal"
+          title="新增数据源"
```
Comment (caosm) [inline web/src/components/integration/component/List.js]: i18n

File: web/src/store/integration.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+useStrict(true);
+
+class Integration {
+  @observable integrationList = [];
+  @action.bound
+  getIntegrationList(workflowId) {
+    this.integrationList = [
+      {
+        id: '5c04e7a73c17eb00019e5a32',
```
Comment (caosm) [inline web/src/store/integration.js]: remove mockdata?

File: web/src/locale/zh-CN.json [hunk 4]
```diff
@@ -13,11 +13,24 @@
...
+    "integration": "集成中心"
   },
   "workflow": {
     "recentVersion": "最近版本",
     "creator": "创建者",
     "creationTime": "创建时间"
+  },
+  "integration": {
+    "type": "类型",
+    "integrationTime": "集成时间",
```
Comment (supereagle) [inline web/src/locale/zh-CN.json]: creationTime: 创建时间

**Pull request merged**